Editor/Processors/SkinnedMeshes/InternalAutoFreezeMeaninglessBlendShapeProcessor.cs

```diff
@@ -35,35 +40,63 @@ public override void Process(BuildContext context, MeshInfo2 target)
                 value[vertex.BlendShapeBufferVertexIndex] = true;
             }
 
+            Profiler.BeginSample("buffers loop");
+            
+            var shapeCount = buffers.Sum(x => x.Key.Shapes.Count);
+            
+            NativeArray<bool> isMeaningfulArray = new NativeArray<bool>(shapeCount, Allocator.TempJob);
```

Comment:
```suggestion
            using var isMeaningfulArray = new NativeArray<bool>(shapeCount, Allocator.TempJob);
```

not disposing will leak and create warning / error message 4 frames later so we should dispose with `using var` or using statement

Reply:
oops, forgot about that!